Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -227,6 +245,8 @@ extension ActivityStreamPanel {
             return configureTopSitesCell(cell, forIndexPath: indexPath)
         case .Highlights:
             return configureHistoryItemCell(cell, forIndexPath: indexPath)
+        case .HighlightIntro:
+            return configureHighlightIntroCell(cell, forIndexPath: indexPath)
```

Comment:
if this function is that short, you don't need to make one? Unless you want that formality